src/App.js

```diff
@@ -1,9 +1,8 @@
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
-import './App.css';
 import { useEffect } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
 import { fetchGetBooks } from './redux/books/booksSlice';
-
+import './app.css';
```

Comment:
- Hi, There is a typo mistake you could fix here. I would suggest you to change `app.css` to `App.css`
![Capture](https://user-images.githubusercontent.com/63972670/224365994-da4f5374-ba3a-4f66-b717-0df985dd11b6.PNG)